themes/Backend/ExtJs/backend/order/controller/document.js

```diff
@@ -60,13 +60,27 @@ Ext.define('Shopware.apps.Order.controller.Document', {
     },
 
     /**
-     * Loads a new mail and open a new mail window
+     * Opens the window to send an email with a selected document to the customer.
+     *
+     * @param { Shopware.apps.Order.model.Receipt } record
      */
     openMail: function(record) {
-        var me = this,
-            order = me.getDocumentWindow().record;
+        var order = this.getDocumentWindow().record;
```

Comment:
We probably shouldn't remove the `me` alias since that seems to be a shopware-specific convention.